### PR TITLE
24107 Added designations for continued in business types

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#24107

*Description of changes:*
- app version = 5.5.10
- added designations for continued in business types (C, CBEN, CCC, CUL)
- imported latest shared enums

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-779-en1w3lo6.web.app

**Comment by severinbeauvais** [comment]
### Screenshots

Name Change - CCC
image

Name Change - ULC
image

Restore - C
image

Restore - CBEN
image

**Comment by severinbeauvais** [comment]
Do not merge yet. I will update https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/name-request-codes.ts and import the latest enums package in Namerequest UI as part of this PR.

File: app/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "@babel/compat-data": "^7.24.4",
     "@bcrs-shared-components/breadcrumb": "2.1.24",
     "@bcrs-shared-components/corp-type-module": "1.0.15",
-    "@bcrs-shared-components/enums": "1.1.7",
+    "@bcrs-shared-components/enums": "1.1.13",
```
Comment (severinbeauvais) [inline app/package.json:20]: This is to pull in the latest NR request type codes (as added in bcgov/entity#23382).

These codes are not used in Namerequest UI but it's good to keep things up to date.

File: app/src/list-data/designations.ts [hunk 2]
```diff
@@ -27,6 +27,8 @@ const CooperativeDesignations: DesignationI = {
 
 export const Designations = {
   BC: CorporateDesignations,
+  C: CorporateDesignations, // continued in BC Limited Company
```
Comment (severinbeauvais) [inline app/src/list-data/designations.ts:30]: C = same as CR

File: app/src/list-data/designations.ts [hunk 3]
```diff
@@ -27,6 +27,8 @@ const CooperativeDesignations: DesignationI = {
...
+  CBEN: CorporateDesignations, // continued in Benefit Company
```
Comment (severinbeauvais) [inline app/src/list-data/designations.ts:31]: CBEN = same as BC (Benefit Company)

File: app/src/list-data/designations.ts [hunk 4]
```diff
@@ -35,8 +37,23 @@ export const Designations = {
...
     end: true
   },
+  CCC: { // continued in Community Contribution Company
+    words: [
+      'CCC',
+      'COMMUNITY CONTRIBUTION COMPANY',
+      ...CorporateDesignations.words
+    ],
+    end: true
+  },
```
Comment (severinbeauvais) [inline app/src/list-data/designations.ts:47]: CCC = same as CC

File: app/src/list-data/designations.ts [hunk 5]
```diff
@@ -35,8 +37,23 @@ export const Designations = {
...
   CP: CooperativeDesignations,
   CR: CorporateDesignations,
+  CUL: { // continued in Unlimited Liability Company
+    words: [
+      'ULC',
+      'UNLIMITED LIABILITY COMPANY'
+    ],
+    end: true
+  },
```
Comment (severinbeauvais) [inline app/src/list-data/designations.ts:56]: CUL = same as ULC

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-779-en1w3lo6.web.app

**Comment by severinbeauvais** [comment]
This is ready for your review, and I'd love to get this merged and into Dev asap (to unblock further testing). Thanks!

**Review by eve-git — APPROVED** [review]
LGTM

**Pull request merged**